src/odb/include/odb/db.h

```diff
@@ -983,6 +983,16 @@ class dbBlock : public dbObject
   void addBTermConstraintByDirection(dbIoType direction,
                                      const Rect& constraint_region);
 
+  ///
+  /// Add region constraint for dbBTerms according to their names given a the
+  /// dimensions of the constraint region.
+  ///
+  void addBTermsToConstraint(const std::vector<dbBTerm*>& bterms,
```

Comment:
Is this just for tcl?  Its not clear why we need an overload

Reply:
Yes, it is for Tcl. I can't create the odb::Rect in the Tcl code, AFAIK.

Reply:
```
    set rect [odb::Rect]
    $rect init $start $start $end $end
```
from src/odb/test/create_sboxes.tcl.  I'm not sure why you can't directly pass args to the ctor (yet).

Reply:
Thanks! I'll remove the overload.